src/components/action-buttons.component.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
```

Comment:
I called this "action buttons" and not "dispense button", because I intend to add the "pause" and "cancel/remove" buttons here that are part of upcoming tickets.